de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/ModelAccessResourceTest.java

```diff
@@ -0,0 +1,432 @@
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.server.resources;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import java.io.StringWriter;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.xml.bind.JAXBException;
+
+import org.eclipse.emf.transaction.RecordingCommand;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.Before;
+import org.junit.Test;
+
+import de.dlr.sc.virsat.model.concept.list.IBeanList;
+import de.dlr.sc.virsat.model.concept.types.IBeanObject;
+import de.dlr.sc.virsat.model.concept.types.category.IBeanCategoryAssignment;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyBoolean;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyComposed;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyEnum;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyFloat;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyInt;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyReference;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyResource;
+import de.dlr.sc.virsat.model.concept.types.property.BeanPropertyString;
+import de.dlr.sc.virsat.model.dvlm.concepts.Concept;
+import de.dlr.sc.virsat.model.dvlm.json.JAXBUtility;
+import de.dlr.sc.virsat.model.dvlm.structural.StructuralElementInstance;
+import de.dlr.sc.virsat.model.extension.tests.model.TestCategoryAllProperty;
+import de.dlr.sc.virsat.model.extension.tests.model.TestCategoryBeanA;
+import de.dlr.sc.virsat.model.extension.tests.model.TestCategoryComposition;
+import de.dlr.sc.virsat.model.extension.tests.model.TestCategoryCompositionArray;
+import de.dlr.sc.virsat.model.extension.tests.model.TestCategoryIntrinsicArray;
+import de.dlr.sc.virsat.model.extension.tests.model.TestCategoryReference;
+import de.dlr.sc.virsat.model.extension.tests.model.TestCategoryReferenceArray;
+import de.dlr.sc.virsat.model.extension.tests.model.TestStructuralElement;
+import de.dlr.sc.virsat.project.editingDomain.VirSatTransactionalEditingDomain;
+import de.dlr.sc.virsat.project.resources.VirSatResourceSet;
+import de.dlr.sc.virsat.server.test.AServerRepositoryTest;
+
+public class ModelAccessResourceTest extends AServerRepositoryTest {
+
+	private Concept conceptTest;
+	private VirSatResourceSet resourceSet;
+	
+	private TestStructuralElement tSei;
+	
+	private TestCategoryAllProperty tcAllProperty;
+	private TestCategoryBeanA tcBeanA;
+	private TestCategoryComposition tcComposition;
+	private TestCategoryReference tcReference;
+	private TestCategoryIntrinsicArray tcIntrinsicArray;
+	private TestCategoryCompositionArray tcCompositionArray;
+	private TestCategoryReferenceArray tcReferenceArray;
+	
+	private BeanPropertyString tString;
```

Comment:
Hmm I guess the t stands for test, but that isnt a 100% clear. Maybe we can rename these to beanString, etc? In the above variables I think its fine because the "Test" actually part of the class name. So when I see tString I first thought we somehow have a class TestString. So this might be a bit misleading.

Reply:
Renamed to `beanString`